scripts/restart-server

```diff
@@ -201,10 +202,15 @@ if has_application_server():
         )
         if uwsgi_status.returncode == 0:
             logging.info("Starting rolling restart of django server")
+            with contextlib.suppress(FileNotFoundError):
+                os.unlink("/var/lib/zulip/django-workers.ready")
             with open("/home/zulip/deployments/uwsgi-control", "w") as control_socket:
                 # "c" is chain-reloading:
                 # https://uwsgi-docs.readthedocs.io/en/latest/MasterFIFO.html#available-commands
                 control_socket.write("c")
+            while not os.path.exists("/var/lib/zulip/django-workers.ready"):
+                time.sleep(1)
```

Comment:
Do we want a print each time we re-sleep? I usually like that sort of active indicator of what's happening during delays.

Reply:
Sure.